src/cli/bootstrap/bootstrap.ts

```diff
@@ -286,6 +286,16 @@ async function createTonomyApps(newPublicKey: PublicKey, newSigner: Signer): Pro
         publicKey: newPublicKey,
         signer,
     });
+
+    await createApp({
+        appName: `${settings.config.ecosystemName} Website`,
+        usernamePrefix: 'tonomy-console',
+        description: `${settings.config.ecosystemName} website to manager your ID and Data`,
```

Comment:
`Developer console to manage ${settings.config.ecosystemName} applications and infrastucture`